azure-pipelines.yml

```diff
@@ -231,14 +231,11 @@ stages:
           demands: ImageOverride -equals windows.vs2022.amd64
         variables:
           - group: Publish-Build-Assets
-          - group: DotNetBot-GitHub
         steps:
           - checkout: self
             clean: true
           - powershell: eng/create-baridtag.ps1
-              -azdoToken $(dn-bot-dotnet-build-rw-code-rw)
```

Comment:
Were these other tokens just overkill for auth?

Reply:
Seems to be, yeah. Which I guess makes sense for `darc get-asset` as it should just be a query to the BAR database. 

I removed it to try to make testing easier but didn't ultimately use it.

Reply:
I mean, if it's not necessary, no reason to keep them around :)